src/app/app.component.html

```diff
@@ -18,36 +18,48 @@
       </md-list>
     </md-content>
 
-    <md-content layout-padding>
+    <md-content layout-padding *ngIf="walletInitialized">
       <md-accordion displayMode="flat" [multi]="'multi'" class="md-expansion-menu">
         <md-expansion-panel #panel1 expanded="true">
           <md-expansion-panel-header>Wallet</md-expansion-panel-header>
           <md-content layout-padding class="submenuarea">
             <md-list class="sidesubmenu">
-              <a routerLinkActive="{{ walletInitialized ? 'active' : null }}" [routerLink]="walletInitialized ? '/wallet/receive' : null">
+              <a routerLinkActive="active" [routerLink]="'/wallet/receive'">
                 <md-list-item>
                   <md-icon fontSet="ncIcon" fontIcon="nc-simple-right"></md-icon>
                   <span class="menu-item">Receive</span>
                 </md-list-item>
               </a>
-              <a routerLinkActive="{{ walletInitialized ? 'active' : null }}" [routerLink]="walletInitialized ? '/wallet/send' : null">
+              <a routerLinkActive="active" [routerLink]="'/wallet/send'">
                 <md-list-item>
                   <md-icon fontSet="ncIcon" fontIcon="nc-simple-left"></md-icon>
                   <span class="menu-item">Send</span>
                 </md-list-item>
               </a>
-              <a routerLinkActive="{{ walletInitialized ? 'active' : null }}" [routerLink]="walletInitialized ? '/wallet/history' : null">
+              <a routerLinkActive="active" [routerLink]="'/wallet/history'">
                 <md-list-item>
                   <md-icon fontSet="ncIcon" fontIcon="nc-time-3"></md-icon>
                   <span class="menu-item">History</span>
                 </md-list-item>
               </a>
-              <a routerLinkActive="{{ walletInitialized ? 'active' : null }}" [routerLink]="walletInitialized ? '/wallet/address-book' : null">
+              <a routerLinkActive="active" [routerLink]="'/wallet/address-book'">
                 <md-list-item>
                   <md-icon fontSet="ncIcon" fontIcon="nc-multiple-19"></md-icon>
                   <span class="menu-item">Address Book</span>
                 </md-list-item>
               </a>
+            </md-list>
+          </md-content>
+        </md-expansion-panel>
+      </md-accordion>
+    </md-content>
+
+    <md-content layout-padding *ngIf="walletInitialized === false">
```

Comment:
You might want to add an additional wallet after you've added a wallet I use 2 wallets...

Reply:
Using "add account" will completely mess up the UX because we'd have to keep track of which HD seed we're on (address generation, no separation of outputs, the list goes on). I think multiwallet support will give the users exactly what they need.